.travis.yml

```diff
@@ -117,4 +121,5 @@ cache:
     - validator/node_modules
     - validator/nodejs/node_modules
     - validator/webui/node_modules
+    - validator/java/bin
```

Comment:
nit: Maybe rename this directory to `bazel_bin` so future maintainers know exactly what it is for?

Reply:
Also, curious: How long does this add to the caching step? (You'll need to check once it hits `master`.)

Reply:
SGTM. If it's just the one `.deb` file as you confirmed, this should not affect overall caching time.